src/Protocol/Diaspora/Factory/DiasporaContact.php

```diff
@@ -74,7 +74,7 @@ public function createfromProbeData(array $data, int $uriId, \DateTime $created,
 		return new \Friendica\Protocol\Diaspora\Entity\DiasporaContact(
 			new Uri($data['url']),
 			$created,
-			$data['guid'],
+			$data['guid'] ?? null,
```

Comment:
Not having a guid is a serious problem for the DiasporaContact since all communication is done via the guid. Weh should have a look why there is no guid in some cases.